src/app/Settings/AutoRefresh.tsx

```diff
@@ -104,9 +106,13 @@ const Component = () => {
 };
 
 export const AutoRefresh: UserSetting = {
-  title: 'Auto-Refresh',
-  description:
-    'Set the refresh period for content views. Views normally update dynamically via WebSocket notifications, so this should not be needed unless WebSockets are not working.',
+  titleKey: 'SETTINGS.AUTO_REFRESH.TITLE',
+  descConstruct: 'SETTINGS.AUTO_REFRESH.DESCRIPTION',
   content: Component,
-  category: 'Connectivity',
+  category: 'SETTINGS.CATEGORIES.CONNECTIVITY',
 };
+
+/**
+ * t('SETTINGS.AUTO_REFRESH.TITLE')
+ * t('SETTINGS.AUTO_REFRESH.DESCRIPTION')
+ */
```

Comment:
Remove comment?

Reply:
Ahhh that was some limitation of the i18next parser that we are using. It just could not parse dynamic keys (must be explicit). If we remove these, it will remove them from the json file next time we run `yarn localize`.

https://github.com/i18next/i18next-parser#caveats

So, I have been including them...Not very nice tho.

Reply:
Ohh yes good idea! Maybe we can just use `i18n.ts` for all files under the same top-level directory under `/src`?

Reply:
So in this case a `src/app/Settings/i18n.ts`? That would work too, though I think it might be more easily overlooked when changes are made to specific component files if it's done that way.

Reply:
Yes!! I am just a bit concerned that if we have new separate files for all component files that has a couple comment lines, for example under `/Settings`..., it might be quite clustered?